test/FunctionalTests/Microsoft.OData.Edm.Tests/Csdl/Serialization/EdmModelCsdlSerializationVisitorTests.cs

```diff
@@ -815,6 +815,34 @@ public void VerifyTermWrittenCorrectly()
 }");
         }
 
+        [Fact]
+        public void VerifyTermWithBaseTermWrittenCorrectly()

```

Comment:
>VerifyTermWithBaseTermWrittenCorrectly [](start = 20, length = 38)

Please add XML Variation as well.

Reply:
Yes. Do you mean "Line 827 - 828"

---
In reply to: [545398455](https://github.com/OData/odata.net/pull/1962#discussion_r545398455) [](ancestors = 545398455)

Reply:
Yes; missed that -- thanks.

---
In reply to: [545399808](https://github.com/OData/odata.net/pull/1962#discussion_r545399808) [](ancestors = 545399808,545398455)